components/home/testimonials.tsx

```
@@ -0,0 +1,145 @@
+                      <div className="bg-primary text-primary-foreground flex h-12 w-12 items-center justify-center rounded-full text-lg font-semibold">
+                        {testimonial.name.charAt(0)}
+                      </div>
+                    )}
+                    <div>
+                      <h3 className="text-foreground text-xl font-semibold">{testimonial.name}</h3>
+                      <p className="text-muted-foreground text-sm">@{testimonial.tag}</p>
+                    </div>
+                  </div>
+                  <p
+                    className="text-foreground [&>a]:text-primary [&>a]:no-underline [&>a]:font-semibold"
+                    dangerouslySetInnerHTML={{ __html: testimonial.description }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)